netbox/dcim/models/racks.py

```diff
@@ -449,6 +450,13 @@ def get_power_utilization(self):
 
         return int(allocated_draw / available_power_total * 100)
 
+    def get_total_weight(self):
```

Comment:
1. We should wrap this with `@cached_property`
2. We probably want to add a similar method to the `Device` model